armada/handlers/armada.py

```diff
@@ -87,19 +87,19 @@ def sync(self):
         '''
 
         # extract known charts on tiller right now
+        if not self.skip_pre_flight:
+            LOG.info("Performing Checking Pre Flight Tasks")
```

Comment:
nit: This could be rephrased to "Performing Pre-Flight Tasks Checking" to make it more clear.